app/Services/Gambit.php

```diff
@@ -53,13 +53,17 @@ public function relaySignup(Signup $signup)
         }
 
         try {
-            $response = $this->client->post('/api/v2/messages?origin=signup', [
+            $this->client->post('/api/v2/messages?origin=signup', [
                 'json' => $payload,
             ]);
         } catch (ClientException $exception) {
-            // We expect to get 422s for any users who sign up for a campaign but don't
-            // have a mobile on their profile. These should not count as failures.
-            if ($response->getStatusCode() !== 422) {
+            $response = $exception->getResponse();
+
+            if ($response->getStatusCode() === 422) {
+                // We expect to get 422s from this endpoint for any users who sign up for
+                // a campaign but don't have a mobile on their profile. These should not
```

Comment:
Another scenario is the user does have a mobile - but they are unsubscribed (`sms_status` is equal to `stop` or `undeliverable`)

Reply:
Oh thanks for noting that! I updated the comment in 145e265.